decidim-elections/config/locales/en.yml

```diff
@@ -531,7 +531,7 @@ en:
             title: Restore election keys for %{election}
             upload_election_keys: Upload election keys
             upload_icon: Icon that indicates a download action
-          tally_steps:
+          tally_started_steps:
```

Comment:
I understand that the change from `tally` to `tally_started` is necessary for Enumerable#tally as discussed at https://github.com/decidim/decidim-bulletin-board/pull/251/commits/4850a5d527642cfc535ad7a1d79756c1bb06e415#r839324288 

So, I understand that it's also necessary to change `tally_steps` to `tally_started_steps`, or this is done to keep consistency between the naming?

Reply:
This is because the partial is dynamically named and referred to here:
https://github.com/decidim/decidim/blob/de7339fcadc5babcfbb929e1b1cf85dfc3f81aa8/decidim-elections/app/views/decidim/elections/trustee_zone/elections/show.html.erb#L18

So, after the `current_step` is changed from `tally` to `tally_started`, this render would break otherwise.

Reply:
Ok, makes sense!